ci/azure-pipelines.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - checkout: self
       - script: make unit-test
         displayName: Run unit test
-      - script: bash <(curl https://codecov.io/bash)
+      - script: bash <(curl https://codecov.io/bash) -t 276dd96d-96a9-4518-b7f7-5028a2ddf658
```

Comment:
can we keep this a secret in the pipeline?

Reply:
I actually remember reading somewhere that it's possible to do that...but I'm having trouble finding the link...

Reply:
Although according to the documentation, Azure Pipelines shouldn't even need a token. It's weird that it doesn't work without it anymore.

https://docs.codecov.io/docs/about-the-codecov-bash-uploader

Reply:
done